validator/src/cli.rs

```diff
@@ -901,6 +902,16 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .hidden(hidden_unless_forced())
                 .help("Controls the rate of the clients connections per IpAddr per minute."),
         )
+        .arg(
+            Arg::with_name("tpu_num_quic_endpoints")
+                .long("tpu-num-quic-endpoints")
+                .takes_value(true)
+                .default_value(&default_args.tpu_num_quic_endpoints)
+                .validator(is_parsable::<usize>)
+                .hidden(hidden_unless_forced())
+                .help("Control the number of TPU QUIC endpoints. Higher QUIC endpoints offers better \
+                       throughput at the higher CPU cost"),
```

Comment:
This is a bit misleading because it's both TPU and TPU-FWD? I'd probably call
the option "num-quic-endpoints" and the help I'd say:

"The number of QUIC endpoints used for TPU and TPU-FWD. Can be increased to
increase network ingest throughput, at the expense of higher CPU and general
validator load."